master build fails on arch linux

I tried installing current oni2 master from the AUR and also tried a manual git build both failing on esy-skia


<details>
  <summary>Build console log</summary>

esy bootstrap
info building @esy-ocaml/libffi@github:onivim/libffi#590b041@d41d8cd9
info building @esy-ocaml/substs@0.0.1@d41d8cd9
info building ocaml@4.8.1000@d41d8cd9
info building yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9
info building esy-cmake@github:prometheansacrifice/esy-cmake#2a47392def755@d41d8cd9
info building esy-sdl2@2.0.10006@d41d8cd9
info building @esy-cross/ninja-build@1.8.2001@d41d8cd9
info building esy-nasm@github:revery-ui/esy-nasm#64a802b@d41d8cd9
info building @esy-ocaml/substs@0.0.1@d41d8cd9: done
info building libvim@8.10869.40@d41d8cd9
info building @esy-cross/ninja-build@1.8.2001@d41d8cd9: done
info building esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9
info building esy-nasm@github:revery-ui/esy-nasm#64a802b@d41d8cd9: done
info building esy-tree-sitter@1.4.1@d41d8cd9
info building @esy-ocaml/libffi@github:onivim/libffi#590b041@d41d8cd9: done
info building esy-help2man@github:esy-packages/esy-help2man#c8e6931d1dcf58a81bd801145a777fd3b115c443@d41d8cd9
info building esy-help2man@github:esy-packages/esy-help2man#c8e6931d1dcf58a81bd801145a777fd3b115c443@d41d8cd9: done
info building texinfo@github:esy-packages/esy-texinfo#4a05feafbbcc4c57d5d25899fbdab98961b9a69c@d41d8cd9
info building esy-tree-sitter@1.4.1@d41d8cd9: done
info building @opam/conf-m4@opam:1@3b2b148a
info building @opam/conf-m4@opam:1@3b2b148a: done
info building @opam/base-unix@opam:base@87d0b2eb
info building @opam/base-unix@opam:base@87d0b2eb: done
info building @opam/base-threads@opam:base@36803084
info building @opam/base-threads@opam:base@36803084: done
info building autoconf@github:esy-packages/esy-autoconf#ea58671a08c7ebf89b0cebd41f4950ca67157b01@d41d8cd9
info building autoconf@github:esy-packages/esy-autoconf#ea58671a08c7ebf89b0cebd41f4950ca67157b01@d41d8cd9: done
info building automake@github:esy-packages/esy-automake#f4fecab89169dd64cc363ec902d828c55c257fc0@d41d8cd9
info building automake@github:esy-packages/esy-automake#f4fecab89169dd64cc363ec902d828c55c257fc0@d41d8cd9: done
info building esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9: done
info building libvim@8.10869.40@d41d8cd9: done
info building texinfo@github:esy-packages/esy-texinfo#4a05feafbbcc4c57d5d25899fbdab98961b9a69c@d41d8cd9: done
info building esy-libtools@github:bryphe/esy-libtools#62766a2@d41d8cd9
info building yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9: done
info building @opam/conf-pkg-config@opam:1.1@55d8e330
info building @opam/conf-pkg-config@opam:1.1@55d8e330: done
info building esy-libtools@github:bryphe/esy-libtools#62766a2@d41d8cd9: done
info building esy-libvterm@1.0.0@d41d8cd9
info building esy-libvterm@1.0.0@d41d8cd9: done
info building ocaml@4.8.1000@d41d8cd9: done
info building @opam/dune@opam:2.5.1@a0c1e658
info building @opam/ocamlfind@opam:1.8.1@ff07b0f9
info building @opam/ocamlbuild@opam:0.14.0@6ac75d03
info building @opam/seq@opam:base@d8d7de1d
info building @opam/seq@opam:base@d8d7de1d: done
info building @opam/ocamlfind@opam:1.8.1@ff07b0f9: done
info building @opam/base-bytes@opam:base@19d0c2ff
info building @opam/base-bytes@opam:base@19d0c2ff: done
info building @opam/ocamlbuild@opam:0.14.0@6ac75d03: done
info building @opam/topkg@opam:1.0.1@a42c631e
info building @opam/uchar@opam:0.0.2@c8218eea
info building @opam/uchar@opam:0.0.2@c8218eea: done
info building @opam/topkg@opam:1.0.1@a42c631e: done
info building @opam/react@opam:1.2.1@0e11855f
info building @opam/uutf@opam:1.0.2@4440868f
info building @opam/astring@opam:0.8.3@4e5e17d5
info building @opam/uutf@opam:1.0.2@4440868f: done
info building @opam/uucp@opam:13.0.0@e9b515e0
info building @opam/react@opam:1.2.1@0e11855f: done
info building @opam/astring@opam:0.8.3@4e5e17d5: done
info building @opam/dune@opam:2.5.1@a0c1e658: done
info building @opam/cppo@opam:1.6.6@f4f83858
info building @opam/fix@opam:20200131@0ecd2f01
info building @opam/menhirLib@opam:20200211@93d0f001
info building @opam/menhirSdk@opam:20200211@b2a79ec0
info building @opam/ppx_derivers@opam:1.2.1@ecf0aa45
info building @opam/ppx_derivers@opam:1.2.1@ecf0aa45: done
info building @opam/result@opam:1.5@6b753c82
info building @opam/menhirSdk@opam:20200211@b2a79ec0: done
info building @opam/stdlib-shims@opam:0.1.0@d957c903
info building @opam/result@opam:1.5@6b753c82: done
info building @opam/dune-private-libs@opam:2.5.1@60c1661f
info building @opam/fix@opam:20200131@0ecd2f01: done
info building @opam/mmap@opam:1.1.0@b85334ff
info building @opam/stdlib-shims@opam:0.1.0@d957c903: done
info building @opam/re@opam:1.9.0@d4d5e13d
info building @opam/menhirLib@opam:20200211@93d0f001: done
info building @opam/easy-format@opam:1.3.2@0484b3c4
info building @opam/cppo@opam:1.6.6@f4f83858: done
info building @opam/camomile@opam:1.0.2@51b42ad8
info building @opam/mmap@opam:1.1.0@b85334ff: done
info building @opam/decoders@opam:0.3.0@044a0d64
info building @opam/easy-format@opam:1.3.2@0484b3c4: done
info building @opam/integers@opam:0.3.0@d6eefd3a
info building @opam/decoders@opam:0.3.0@044a0d64: done
info building @opam/ppx_tools@opam:6.0+4.08.0@5f5453f4
info building @opam/integers@opam:0.3.0@d6eefd3a: done
info building @opam/sexplib0@opam:v0.13.0@3f54c2be
info building @opam/re@opam:1.9.0@d4d5e13d: done
info building @opam/ocaml-compiler-libs@opam:v0.12.1@5c34eb0d
info building @opam/sexplib0@opam:v0.13.0@3f54c2be: done
info building @opam/markup@opam:0.8.2@87975241
info building @opam/ocaml-compiler-libs@opam:v0.12.1@5c34eb0d: done
info building @opam/trie@opam:1.0.0@d2efc587
info building @opam/trie@opam:1.0.0@d2efc587: done
info building @opam/psq@opam:0.2.0@247756d4
info building @opam/psq@opam:0.2.0@247756d4: done
info building revery-text-wrap@github:revery-ui/revery-text-wrap#966383e@d41d8cd9
info building @opam/ppx_tools@opam:6.0+4.08.0@5f5453f4: done
info building @opam/ocaml-migrate-parsetree@opam:1.7.2@bc8b618e
info building @opam/markup@opam:0.8.2@87975241: done
info building @opam/fpath@opam:0.7.2@45477b93
info building revery-text-wrap@github:revery-ui/revery-text-wrap#966383e@d41d8cd9: done
info building @opam/ptime@opam:0.8.5@0051d642
info building @opam/fpath@opam:0.7.2@45477b93: done
info building @opam/fmt@opam:0.8.8@01c3a23c
info building @opam/dune-private-libs@opam:2.5.1@60c1661f: done
info building @opam/menhir@opam:20200211@26571604
info building @opam/ptime@opam:0.8.5@0051d642: done
info building @opam/ocplib-endian@opam:1.1@84c1ca88
info building @opam/fmt@opam:0.8.8@01c3a23c: done
info building @opam/merlin-extend@opam:0.5@a5dd7d4b
info building @opam/ocplib-endian@opam:1.1@84c1ca88: done
info building @opam/biniou@opam:1.2.1@d7570399
info building @opam/merlin-extend@opam:0.5@a5dd7d4b: done
info building @opam/ctypes@opam:0.15.1@b0227b2f
info building @opam/biniou@opam:1.2.1@d7570399: done
info building @opam/tyxml@opam:4.4.0@1dca5713
info building @opam/camomile@opam:1.0.2@51b42ad8: done
info building @opam/mew@opam:0.1.0@a74f69d6
info building @opam/mew@opam:0.1.0@a74f69d6: done
info building @opam/lru@github:bryphe/lru:lru.opam#2708c70@d41d8cd9
info building @opam/lru@github:bryphe/lru:lru.opam#2708c70@d41d8cd9: done
info building @opam/dune-configurator@opam:2.5.1@aeb9d8d5
info building @opam/tyxml@opam:4.4.0@1dca5713: done
info building @opam/yojson@github:onivim/yojson:yojson.opam#f480aef@d41d8cd9
info building @opam/dune-configurator@opam:2.5.1@aeb9d8d5: done
info building @opam/charInfo_width@opam:1.1.0@9d8d61b2
info building @opam/charInfo_width@opam:1.1.0@9d8d61b2: done
info building @opam/mew_vi@opam:0.3.0@bf3ebe06
info building @opam/ocaml-migrate-parsetree@opam:1.7.2@bc8b618e: done
info building @opam/junit@opam:2.0.2@0b7bd730
info building @opam/yojson@github:onivim/yojson:yojson.opam#f480aef@d41d8cd9: done
info building @opam/lwt@opam:4.5.0@677655b4
info building @opam/junit@opam:2.0.2@0b7bd730: done
info building @opam/base@opam:v0.13.1@7d937ed0
info building @opam/mew_vi@opam:0.3.0@bf3ebe06: done
info building @opam/zed@opam:3.0.1@44b41979
info building @opam/uucp@opam:13.0.0@e9b515e0: done
info building @opam/ppx_tools_versioned@opam:5.3.0@2c0dc023
info building @opam/zed@opam:3.0.1@44b41979: done
info building @opam/ppxfind@opam:1.4@1e01d2a5
info building @opam/menhir@opam:20200211@26571604: done
info building @opam/decoders-yojson@opam:0.3.0@b9081413
info building @opam/decoders-yojson@opam:0.3.0@b9081413: done
info building @opam/atdgen-runtime@opam:2.2.1@6a3a6395
info building @opam/ppxfind@opam:1.4@1e01d2a5: done
info building @opam/merlin@opam:3.3.4@5fcabf21
info building @opam/atdgen-runtime@opam:2.2.1@6a3a6395: done
info building @opam/ppx_yojson_conv_lib@opam:v0.13.0@97a4cb97
info building @opam/ppx_yojson_conv_lib@opam:v0.13.0@97a4cb97: done
info building @opam/printbox@opam:0.5@82f5d436
info building @opam/printbox@opam:0.5@82f5d436: done
info building @opam/atd@opam:2.2.1@071ab6bd
info building @opam/lwt@opam:4.5.0@677655b4: done
info building @esy-ocaml/reason@3.5.2@d41d8cd9
info building @opam/atd@opam:2.2.1@071ab6bd: done
info building @opam/ppx_deriving@opam:4.4.1@208c6c8f
info building @opam/ppx_deriving@opam:4.4.1@208c6c8f: done
info building @opam/ocaml-lsp-server@github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#04733ed@d41d8cd9
info building @opam/ppx_tools_versioned@opam:5.3.0@2c0dc023: done
info building @opam/logs@opam:0.7.0@1d03143e
info building @opam/base@opam:v0.13.1@7d937ed0: done
info building @opam/lwt_react@opam:1.1.3@72987fcf
info building @opam/lwt_react@opam:1.1.3@72987fcf: done
info building @opam/lwt_log@opam:1.1.1@2d7a797f
info building @opam/lwt_log@opam:1.1.1@2d7a797f: done
info building @opam/atdgen@opam:2.2.1@d73fda11
info building @opam/logs@opam:0.7.0@1d03143e: done
info building @opam/ppx_deriving_yojson@opam:3.5.2@ca415fbe
info building @opam/ppx_deriving_yojson@opam:3.5.2@ca415fbe: done
info building @opam/lwt_ppx@opam:2.0.1@ab0debb8
info building @opam/atdgen@opam:2.2.1@d73fda11: done
info building @opam/stdio@opam:v0.13.0@eb59d879
info building @opam/stdio@opam:v0.13.0@eb59d879: done
info building @opam/lambda-term@opam:3.0.0@50f5dee6
info building @opam/ctypes@opam:0.15.1@b0227b2f: done
info building @opam/ppxlib@opam:0.13.0@65a9c7cc
info building @opam/lwt_ppx@opam:2.0.1@ab0debb8: done
info building @opam/luv@opam:0.5.2@bfb5a32a
info building @opam/merlin@opam:3.3.4@5fcabf21: done
info building @esy-ocaml/reason@3.5.2@d41d8cd9: done
info building rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9
info building reason-font-manager@2.1.1@d41d8cd9
info building rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9: done
info building flex@1.2.3@d41d8cd9
info building @opam/lambda-term@opam:3.0.0@50f5dee6: done
info building reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9
info building reason-font-manager@2.1.1@d41d8cd9: done
info building @reason-native/pastel@0.1.0@d41d8cd9
info building @reason-native/pastel@0.1.0@d41d8cd9: done
info building @reason-native/console@0.0.3@d41d8cd9
info building reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9: done
info building @reason-native/file-context-printer@0.0.3@d41d8cd9
info building @reason-native/console@0.0.3@d41d8cd9: done
info building @reason-native/cli@0.0.1-alpha@d41d8cd9
info building @reason-native/file-context-printer@0.0.3@d41d8cd9: done
info building refmterr@3.3.0@d41d8cd9
info building flex@1.2.3@d41d8cd9: done
info building @reason-native/cli@0.0.1-alpha@d41d8cd9: done
info building @reason-native/rely@3.2.1@d41d8cd9
info building refmterr@3.3.0@d41d8cd9: done
info building reason-libvterm@github:revery-ui/reason-libvterm#91bad1f@d41d8cd9
info building reperf@1.5.0@d41d8cd9
info building reperf@1.5.0@d41d8cd9: done
info building @glennsl/timber@1.0.0@d41d8cd9
info building reason-libvterm@github:revery-ui/reason-libvterm#91bad1f@d41d8cd9: done
info building editor-core-types@github:onivim/editor-core-types#6a8afaf@d41d8cd9
info building @reason-native/rely@3.2.1@d41d8cd9: done
info building editor-input@github:onivim/editor-input#c494950@d41d8cd9
info building @glennsl/timber@1.0.0@d41d8cd9: done
info building isolinear@github:revery-ui/isolinear#b17ce17@d41d8cd9
info building editor-core-types@github:onivim/editor-core-types#6a8afaf@d41d8cd9: done
info building reason-oniguruma@6.94.1002@d41d8cd9
info building isolinear@github:revery-ui/isolinear#b17ce17@d41d8cd9: done
info building reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9
info building editor-input@github:onivim/editor-input#c494950@d41d8cd9: done
info building rench@github:bryphe/rench#a976fe5@d41d8cd9
info building reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9: done
info building reason-jsonrpc@1.5.0@d41d8cd9
info building reason-oniguruma@6.94.1002@d41d8cd9: done
info building reason-libvim@github:onivim/reason-libvim#e791e3a@d41d8cd9
info building rench@github:bryphe/rench#a976fe5@d41d8cd9: done
info building reason-tree-sitter@1.0.1001@d41d8cd9
info building reason-jsonrpc@1.5.0@d41d8cd9: done
info building reason-tree-sitter@1.0.1001@d41d8cd9: done
info building @opam/ocaml-lsp-server@github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#04733ed@d41d8cd9: done
info building reason-libvim@github:onivim/reason-libvim#e791e3a@d41d8cd9: done
info building @opam/ppxlib@opam:0.13.0@65a9c7cc: done
info building @brisk/brisk-reconciler@github:briskml/brisk-reconciler#10cab2d@d41d8cd9
info building @opam/ppx_let@opam:v0.13.0@5703d2be
info building @opam/ppx_let@opam:v0.13.0@5703d2be: done
info building reason-textmate@3.1.2@d41d8cd9
info building @brisk/brisk-reconciler@github:briskml/brisk-reconciler#10cab2d@d41d8cd9: done
info building reason-textmate@3.1.2@d41d8cd9: done
info building esy-sdl2@2.0.10006@d41d8cd9: done
info building @opam/luv@opam:0.5.2@bfb5a32a: done
info building esy-cmake@github:prometheansacrifice/esy-cmake#2a47392def755@d41d8cd9: done
info building reason-sdl2@2.10.3022@d41d8cd9
info building esy-libjpeg-turbo@github:revery-ui/libjpeg-turbo#61e031f@d41d8cd9
info building esy-freetype2@2.9.1007@d41d8cd9
info building esy-harfbuzz@1.9.1008@d41d8cd9
info building esy-macdylibbundler@0.4.5@d41d8cd9
info building esy-macdylibbundler@0.4.5@d41d8cd9: done
info building reason-sdl2@2.10.3022@d41d8cd9: done
info building esy-freetype2@2.9.1007@d41d8cd9: done
info building esy-harfbuzz@1.9.1008@d41d8cd9: done
info building reason-harfbuzz@1.91.8001@d41d8cd9
info building reason-harfbuzz@1.91.8001@d41d8cd9: done
info building esy-libjpeg-turbo@github:revery-ui/libjpeg-turbo#61e031f@d41d8cd9: done
info building esy-skia@github:revery-ui/esy-skia#07c13a9@d41d8cd9
error: build failed with exit code: 1
  build log:
    # esy-build-package: building: esy-skia@github:revery-ui/esy-skia#07c13a9
    # esy-build-package: pwd: /home/mh/.esy/3/b/esy_skia-f124b077
    # esy-build-package: running: 'bash' './esy/build.sh' 'linux' '/home/mh/.esy/3_______________________________________________________________________/i/esy_libjpeg_turbo-fc2602b2'
    buildtools                           @ 505de88083136eefd056e5ee4ca0f01fe9b33de8
    common                               @ 9737551d7a52c3db3262db5856e6bcd62c462b92
    third_party/externals/angle2         @ b001528ffa00e7c15a5002124f707570e59a5697
    third_party/externals/dng_sdk        @ 96443b262250c390b0caefbf3eed8463ba35ecae
    third_party/externals/expat          @ android-6.0.1_r55
    third_party/externals/freetype       @ fa6da7bf6dcbcfa22d2464259ad1cccee3233189
    third_party/externals/googletest     @ dd43b9998e9a44a579a7aba6c1309407d1a5ed95
    third_party/externals/harfbuzz       @ 1.4.2
    third_party/externals/icu            @ ec9c1133693148470ffe2e5e53576998e3650c1d
    third_party/externals/imgui          @ 6384eee34f08cb7eab8d835043e1738e4adcdf75
    third_party/externals/jsoncpp        @ 1.0.0
    third_party/externals/libjpeg-turbo  @ 1.5.3
    third_party/externals/libpng         @ v1.6.33
    third_party/externals/libwebp        @ v0.6.1
    third_party/externals/lua            @ v5-3-4
    third_party/externals/microhttpd     @ 748945ec6f1c67b7efc934ab0808e1d32f2fb98d
    third_party/externals/piex           @ bb217acdca1cc0c16b704669dd6f91a1b509c406
    third_party/externals/rapidjson      @ af223d44f4e8d3772cb1ac0ce8bc2a132b51717f
    third_party/externals/sdl            @ 5d7cfcca344034aff9327f77fc181ae3754e7a90
    third_party/externals/sfntly         @ b18b09b6114b9b7fe6fc2f96d8b15e8a72f66916
    third_party/externals/spirv-headers  @ 661ad91124e6af2272afd00f804d8aa276e17107
    third_party/externals/spirv-tools    @ e9e4393b1c5aad7553c05782acefbe32b42644bd
    third_party/externals/swiftshader    @ cbb80f5f0078a9941f3ec43e83e52c3d15a43bea
    third_party/externals/zlib           @ e7afdfe128e01ca480a28f757b571957befdd962
    llvm toolset-7.0 does not need to be manually activated
    Done. Made 44 targets from 27 files in 58ms
    ninja: Entering directory `/home/mh/.esy/3/b/esy_skia-f124b077/out/Static'
    [1/1206] stamp obj/arm64.stamp
    [2/1206] stamp obj/armv7.stamp
    [3/1206] stamp obj/compile_processors.stamp
    [4/1206] stamp obj/crc32.stamp
    [5/1206] stamp obj/fontmgr_empty.stamp
    [6/1206] stamp obj/fontmgr_fuchsia.stamp
    [7/1206] compile ../../src/ports/SkFontConfigInterface.cpp
    [8/1206] compile ../../src/gpu/GrBackendSurface.cpp
    [9/1206] compile ../../src/effects/Sk1DPathEffect.cpp
    [10/1206] compile ../../src/effects/Sk2DPathEffect.cpp
    [11/1206] compile ../../src/effects/SkColorMatrix.cpp
    [12/1206] compile ../../src/gpu/GrPaint.cpp
    [13/1206] compile ../../src/effects/SkColorFilterImageFilter.cpp
    [14/1206] compile ../../src/gpu/GrAuditTrail.cpp
    [15/1206] compile ../../src/gpu/GrPath.cpp
    [16/1206] compile ../../src/effects/SkColorMatrixFilter.cpp
    [17/1206] compile ../../src/effects/SkDiscretePathEffect.cpp
    [18/1206] compile ../../src/effects/SkAlphaThresholdFilter.cpp
    [19/1206] compile ../../src/effects/SkDashPathEffect.cpp
    [20/1206] compile ../../src/effects/SkComposeImageFilter.cpp
    [21/1206] compile ../../src/effects/SkCornerPathEffect.cpp
    [22/1206] compile ../../src/effects/SkEmbossMask.cpp
    [23/1206] compile ../../src/effects/SkArithmeticImageFilter.cpp
    [24/1206] compile ../../src/effects/SkDropShadowImageFilter.cpp
    [25/1206] compile ../../src/effects/SkEmbossMaskFilter.cpp
    [26/1206] compile ../../src/opts/SkOpts_avx.cpp
    [27/1206] stamp obj/avx.stamp
    [28/1206] compile ../../src/effects/SkImageSource.cpp
    [29/1206] compile ../../src/effects/SkLayerDrawLooper.cpp
    [30/1206] compile ../../src/effects/SkMagnifierImageFilter.cpp
    [31/1206] compile ../../src/effects/SkHighContrastFilter.cpp
    [32/1206] compile ../../src/effects/SkDisplacementMapEffect.cpp
    [33/1206] compile ../../src/effects/SkMergeImageFilter.cpp
    [34/1206] compile ../../src/effects/SkPackBits.cpp
    [35/1206] compile ../../src/effects/SkOffsetImageFilter.cpp
    [36/1206] compile ../../src/effects/SkLumaColorFilter.cpp
    [37/1206] compile ../../src/effects/SkMatrixConvolutionImageFilter.cpp
    [38/1206] compile ../../src/effects/SkPaintFlagsDrawFilter.cpp
    [39/1206] compile ../../src/effects/SkPaintImageFilter.cpp
    [40/1206] compile ../../src/effects/SkOverdrawColorFilter.cpp
    [41/1206] compile ../../src/effects/SkTableMaskFilter.cpp
    [42/1206] compile ../../src/effects/SkPictureImageFilter.cpp
    [43/1206] compile ../../src/effects/SkTileImageFilter.cpp
    [44/1206] compile ../../src/effects/SkShaderMaskFilter.cpp
    [45/1206] compile ../../src/effects/SkTrimPathEffect.cpp
    [46/1206] compile ../../src/shaders/gradients/SkGradientBitmapCache.cpp
    [47/1206] compile ../../src/effects/SkToSRGBColorFilter.cpp
    [48/1206] compile ../../src/effects/SkLightingImageFilter.cpp
    [49/1206] compile ../../src/effects/SkMorphologyImageFilter.cpp
    [50/1206] compile ../../src/effects/SkTableColorFilter.cpp
    [51/1206] compile ../../src/shaders/gradients/Sk4fGradientBase.cpp
    [52/1206] compile ../../src/effects/SkXfermodeImageFilter.cpp
    [53/1206] compile ../../src/ports/SkGlobalInitialization_default.cpp
    [54/1206] compile ../../src/shaders/SkPerlinNoiseShader.cpp
    [55/1206] stamp obj/fontmgr_custom.stamp
    [56/1206] compile ../../src/shaders/gradients/SkRadialGradient.cpp
    [57/1206] compile ../../src/shaders/gradients/Sk4fLinearGradient.cpp
    [58/1206] compile ../../src/ports/SkFontMgr_android_factory.cpp
    [59/1206] compile ../../src/shaders/gradients/SkLinearGradient.cpp
    [60/1206] compile ../../src/shaders/gradients/SkTwoPointConicalGradient_gpu.cpp
    [61/1206] compile ../../src/shaders/gradients/SkTwoPointConicalGradient.cpp
    [62/1206] compile ../../src/ports/SkFontConfigInterface_direct_factory.cpp
    [63/1206] compile ../../src/shaders/gradients/SkSweepGradient.cpp
    [64/1206] compile ../../src/ports/SkFontMgr_fontconfig_factory.cpp
    [65/1206] compile ../../src/ports/SkFontMgr_android_parser.cpp
    [66/1206] compile ../../src/gpu/GrAHardwareBufferImageGenerator.cpp
    [67/1206] compile ../../src/ports/SkFontConfigInterface_direct.cpp
    [68/1206] compile ../../src/gpu/GrBlend.cpp
    [69/1206] compile ../../src/ports/SkFontMgr_FontConfigInterface.cpp
    [70/1206] compile ../../src/ports/SkFontMgr_android.cpp
    [71/1206] compile ../../src/shaders/gradients/SkGradientShader.cpp
    [72/1206] stamp obj/effects.stamp
    [73/1206] compile ../../src/pdf/SkPDFConvertType1FontStream.cpp
    [74/1206] compile ../../src/ports/SkFontMgr_fontconfig.cpp
    [75/1206] compile ../../src/gpu/GrBitmapTextureMaker.cpp
    [76/1206] compile ../../src/gpu/GrBuffer.cpp
    [77/1206] compile ../../src/gpu/GrBackendTextureImageGenerator.cpp
    [78/1206] compile ../../src/pdf/SkPDFFormXObject.cpp
    [79/1206] compile ../../src/pdf/SkPDFResourceDict.cpp
    [80/1206] compile ../../src/gpu/GrBlurUtils.cpp
    [81/1206] compile ../../src/pdf/SkPDFGraphicState.cpp
    [82/1206] compile ../../src/pdf/SkPDFDocument.cpp
    [83/1206] compile ../../src/pdf/SkPDFMakeCIDGlyphWidthsArray.cpp
    [84/1206] stamp obj/run_sksllex.stamp
    [85/1206] compile ../../src/pdf/SkPDFMakeToUnicodeCmap.cpp
    [86/1206] compile ../../src/pdf/SkPDFMetadata.cpp
    [87/1206] compile ../../src/pdf/SkPDFGradientShader.cpp
    [88/1206] compile ../../src/pdf/SkPDFFont.cpp
    [89/1206] compile ../../src/core/SkColorSpaceXform_skcms.cpp
    [90/1206] compile ../../src/c/sk_colortable.cpp
    [91/1206] compile ../../src/pdf/SkPDFUtils.cpp
    [92/1206] compile ../../src/c/sk_data.cpp
    [93/1206] compile ../../src/c/sk_document.cpp
    [94/1206] compile ../../src/pdf/SkPDFDevice.cpp
    [95/1206] compile ../../src/codec/SkIcoCodec.cpp
    [96/1206] compile ../../src/c/sk_drawable.cpp
    [97/1206] compile ../../src/pdf/SkPDFTypes.cpp
    [98/1206] compile ../../src/pdf/SkPDFShader.cpp
    [99/1206] compile ../../src/c/sk_enums.cpp
    [100/1206] compile ../../src/c/sk_general.cpp
    [101/1206] compile ../../src/c/sk_canvas.cpp
    [102/1206] compile ../../src/c/sk_colorfilter.cpp
    [103/1206] compile ../../src/c/sk_codec.cpp
    [104/1206] compile ../../src/c/sk_mask.cpp
    [105/1206] compile ../../src/c/sk_image.cpp
    [106/1206] compile ../../src/c/sk_imagefilter.cpp
    [107/1206] compile ../../src/c/sk_maskfilter.cpp
    [108/1206] compile ../../src/c/sk_matrix.cpp
    [109/1206] compile ../../src/c/sk_paint.cpp
    [110/1206] compile ../../src/c/sk_patheffect.cpp
    [111/1206] compile ../../src/c/sk_picture.cpp
    [112/1206] compile ../../src/c/sk_region.cpp
    [113/1206] compile ../../src/c/sk_path.cpp
    [114/1206] compile ../../src/c/sk_pixmap.cpp
    [115/1206] compile ../../src/c/sk_rrect.cpp
    [116/1206] compile ../../src/codec/SkRawCodec.cpp
    [117/1206] compile ../../src/c/sk_shader.cpp
    [118/1206] compile ../../src/c/sk_stream.cpp
    [119/1206] compile ../../src/c/sk_bitmap.cpp
    [120/1206] compile ../../src/gpu/GrCaps.cpp
    [121/1206] compile ../../src/gpu/GrColorSpaceInfo.cpp
    [122/1206] compile ../../src/gpu/GrColorSpaceXform.cpp
    [123/1206] compile ../../src/gpu/GrDefaultGeoProcFactory.cpp
    [124/1206] compile ../../src/gpu/GrBufferAllocPool.cpp
    [125/1206] compile ../../src/gpu/GrDistanceFieldGenFromVector.cpp
    [126/1206] compile ../../src/gpu/GrDDLContext.cpp
    [127/1206] compile ../../src/gpu/GrDriverBugWorkarounds.cpp
    [128/1206] compile ../../src/gpu/GrClipStackClip.cpp
    [129/1206] compile ../../src/gpu/GrDirectContext.cpp
    [130/1206] compile ../../src/gpu/GrDrawOpAtlas.cpp
    [131/1206] compile ../../src/gpu/GrDrawOpTest.cpp
    [132/1206] compile ../../src/gpu/GrMemoryPool.cpp
    [133/1206] compile ../../src/gpu/GrGpuResourceRef.cpp
    [134/1206] compile ../../src/gpu/GrDrawingManager.cpp
    [135/1206] compile ../../src/gpu/GrContext.cpp
    [136/1206] compile ../../src/gpu/GrFixedClip.cpp
    [137/1206] compile ../../src/gpu/GrGpu.cpp
    [138/1206] compile ../../src/gpu/GrFragmentProcessor.cpp
    [139/1206] compile ../../src/gpu/GrGpuCommandBuffer.cpp
    [140/1206] compile ../../src/gpu/GrGpuResource.cpp
    [141/1206] compile ../../src/gpu/GrImageTextureMaker.cpp
    [142/1206] compile ../../src/gpu/GrOpFlushState.cpp
    [143/1206] compile ../../src/gpu/GrOpList.cpp
    [144/1206] compile ../../src/gpu/effects/GrAARectEffect.cpp
    [145/1206] compile ../../src/gpu/effects/GrArithmeticFP.cpp
    [146/1206] compile ../../src/gpu/effects/GrBlurredEdgeFragmentProcessor.cpp
    [147/1206] compile ../../src/gpu/effects/GrAlphaThresholdFragmentProcessor.cpp
    [148/1206] compile ../../src/gpu/effects/GrCircleBlurFragmentProcessor.cpp
    [149/1206] compile ../../src/gpu/GrPrimitiveProcessor.cpp
    [150/1206] compile ../../src/gpu/GrPathProcessor.cpp
    [151/1206] compile ../../src/gpu/GrPathRendering.cpp
    [152/1206] compile ../../src/gpu/GrPathUtils.cpp
    [153/1206] compile ../../src/gpu/GrPathRenderer.cpp
    [154/1206] compile ../../src/gpu/GrPathRendererChain.cpp
    [155/1206] compile ../../src/gpu/GrRectanizer_pow2.cpp
    [156/1206] compile ../../src/gpu/GrOnFlushResourceProvider.cpp
    [157/1206] compile ../../src/gpu/GrProcessorSet.cpp
    [158/1206] compile ../../src/gpu/GrProcessorUnitTest.cpp
    [159/1206] compile ../../src/gpu/GrRectanizer_skyline.cpp
    [160/1206] compile ../../src/gpu/GrQuad.cpp
    [161/1206] compile ../../src/gpu/GrPipeline.cpp
    [162/1206] compile ../../src/gpu/GrProgramDesc.cpp
    [163/1206] compile ../../src/gpu/GrProcessor.cpp
    [164/1206] compile ../../src/gpu/GrProcessorAnalysis.cpp
    [165/1206] compile ../../src/gpu/GrProxyProvider.cpp
    [166/1206] compile ../../src/gpu/GrRenderTargetProxy.cpp
    [167/1206] compile ../../src/gpu/gl/GrGLInterface.cpp
    [168/1206] compile ../../src/gpu/GrResourceAllocator.cpp
    [169/1206] compile ../../src/gpu/GrRenderTarget.cpp
    [170/1206] compile ../../src/gpu/GrShaderCaps.cpp
    [171/1206] compile ../../src/gpu/GrStencilAttachment.cpp
    [172/1206] compile ../../src/gpu/GrResourceProvider.cpp
    [173/1206] compile ../../src/gpu/GrShape.cpp
    [174/1206] compile ../../src/gpu/GrRenderTargetOpList.cpp
    [175/1206] compile ../../src/gpu/GrResourceCache.cpp
    [176/1206] compile ../../src/gpu/GrReducedClip.cpp
    [177/1206] compile ../../src/gpu/gl/GrGLPath.cpp
    [178/1206] compile ../../src/gpu/GrStyle.cpp
    [179/1206] compile ../../src/gpu/GrStencilSettings.cpp
    FAILED: obj/src/gpu/gpu.GrStencilSettings.o 
    clang++ -MMD -MF obj/src/gpu/gpu.GrStencilSettings.o.d -DNDEBUG -DSK_SAMPLES_FOR_X -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/home/mh/.esy/3_______________________________________________________________________/i/esy_libjpeg_turbo-fc2602b2/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/gpu/GrStencilSettings.cpp -o obj/src/gpu/gpu.GrStencilSettings.o
    ../../src/gpu/GrStencilSettings.cpp:58:31: error: bitwise operation between different enumeration types ('GrStencilSettings::(anonymous enum at ../../src/gpu/GrStencilSettings.h:117:5)' and 'GrStencilFlags') [-Werror,-Wanon-enum-enum-conversion]
        if ((kInvalid_PrivateFlag | kDisabled_StencilFlag) & fFlags) {
             ~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~
    ../../src/gpu/GrStencilSettings.cpp:71:31: error: bitwise operation between different enumeration types ('GrStencilSettings::(anonymous enum at ../../src/gpu/GrStencilSettings.h:117:5)' and 'GrStencilFlags') [-Werror,-Wanon-enum-enum-conversion]
        if ((kInvalid_PrivateFlag | kDisabled_StencilFlag) & (fFlags | that.fFlags)) {
             ~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~
    2 errors generated.
    [180/1206] compile ../../src/gpu/gl/GrGLPathRendering.cpp
    [181/1206] compile ../../src/gpu/GrSKSLPrettyPrint.cpp
    [182/1206] compile ../../src/gpu/GrSurfaceProxyRef.cpp
    [183/1206] compile ../../src/gpu/GrShaderVar.cpp
    [184/1206] compile ../../src/gpu/GrRenderTargetContext.cpp
    [185/1206] compile ../../src/gpu/GrTestUtils.cpp
    [186/1206] compile ../../src/gpu/GrTextureOpList.cpp
    [187/1206] compile ../../src/gpu/GrSoftwarePathRenderer.cpp
    [188/1206] compile ../../src/gpu/GrTessellator.cpp
    ninja: build stopped: subcommand failed.
    error: command failed: 'bash' './esy/build.sh' 'linux' '/home/mh/.esy/3_______________________________________________________________________/i/esy_libjpeg_turbo-fc2602b2' (exited with 1)
    esy-build-package: exiting with errors above...
    
  building esy-skia@github:revery-ui/esy-skia#07c13a9
esy: exiting due to errors above
</details>

--- a/bench.esy.lock/index.json
+++ b/bench.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "41954cf4d697932cc08cce97af63383e",
+  "checksum": "74a66d9a89023d7c756b7f39c2d27ff3",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -131,20 +131,20 @@
       },
       "overrides": [],
       "dependencies": [
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "reason-libvterm@github:revery-ui/reason-libvterm#91bad1f@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#b17ce17@d41d8cd9",
         "@glennsl/timber@1.0.0@d41d8cd9"
       ],
       "devDependencies": []
     },
-    "revery@github:revery-ui/revery#e108530@d41d8cd9": {
-      "id": "revery@github:revery-ui/revery#e108530@d41d8cd9",
+    "revery@github:revery-ui/revery#9f725e0@d41d8cd9": {
+      "id": "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
       "name": "revery",
-      "version": "github:revery-ui/revery#e108530",
+      "version": "github:revery-ui/revery#9f725e0",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/revery#e108530" ]
+        "source": [ "github:revery-ui/revery#9f725e0" ]
       },
       "overrides": [],
       "dependencies": [
@@ -152,8 +152,8 @@
         "reperf@1.5.0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
-        "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
-        "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
+        "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-harfbuzz@1.91.8001@d41d8cd9",
         "reason-font-manager@2.1.1@d41d8cd9", "flex@1.2.3@d41d8cd9",
         "@reason-native/rely@3.2.1@d41d8cd9",
@@ -327,17 +327,17 @@
       ],
       "devDependencies": []
     },
-    "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9": {
-      "id": "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
+    "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9": {
+      "id": "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
       "name": "reason-skia",
-      "version": "github:revery-ui/reason-skia#2352b85",
+      "version": "github:revery-ui/reason-skia#6b459c7",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/reason-skia#2352b85" ]
+        "source": [ "github:revery-ui/reason-skia#6b459c7" ]
       },
       "overrides": [],
       "dependencies": [
-        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "esy-skia@github:revery-ui/esy-skia#07c13a9@d41d8cd9",
         "esy-sdl2@2.0.10006@d41d8cd9", "esy-freetype2@2.9.1007@d41d8cd9",
@@ -347,14 +347,14 @@
       ],
       "devDependencies": [ "@opam/dune@opam:2.5.1@a0c1e658" ]
     },
-    "reason-sdl2@2.10.3022@d41d8cd9": {
-      "id": "reason-sdl2@2.10.3022@d41d8cd9",
+    "reason-sdl2@2.10.3023@d41d8cd9": {
+      "id": "reason-sdl2@2.10.3023@d41d8cd9",
       "name": "reason-sdl2",
-      "version": "2.10.3022",
+      "version": "2.10.3023",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3022.tgz#sha1:9987104b0bad2627e66c4632e8525a177fdb8a2d"
+          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3023.tgz#sha1:ee3fea956160c9139ed588acf76c02bd80c4c962"
         ]
       },
       "overrides": [],
@@ -1179,13 +1179,13 @@
       "overrides": [ "bench.json" ],
       "dependencies": [
         "revery-terminal@github:revery-ui/revery-terminal#4e8ca1c@d41d8cd9",
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "reperf@1.5.0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "refmterr@3.3.0@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9",
         "reason-tree-sitter@1.0.1001@d41d8cd9",
-        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "reason-libvim@github:onivim/reason-libvim#ce6b248@d41d8cd9",
         "reason-jsonrpc@1.5.0@d41d8cd9", "ocaml@4.9.0@d41d8cd9",

--- a/esy.lock/index.json
+++ b/esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "41954cf4d697932cc08cce97af63383e",
+  "checksum": "74a66d9a89023d7c756b7f39c2d27ff3",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -131,20 +131,20 @@
       },
       "overrides": [],
       "dependencies": [
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "reason-libvterm@github:revery-ui/reason-libvterm#91bad1f@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#b17ce17@d41d8cd9",
         "@glennsl/timber@1.0.0@d41d8cd9"
       ],
       "devDependencies": []
     },
-    "revery@github:revery-ui/revery#e108530@d41d8cd9": {
-      "id": "revery@github:revery-ui/revery#e108530@d41d8cd9",
+    "revery@github:revery-ui/revery#9f725e0@d41d8cd9": {
+      "id": "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
       "name": "revery",
-      "version": "github:revery-ui/revery#e108530",
+      "version": "github:revery-ui/revery#9f725e0",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/revery#e108530" ]
+        "source": [ "github:revery-ui/revery#9f725e0" ]
       },
       "overrides": [],
       "dependencies": [
@@ -152,8 +152,8 @@
         "reperf@1.5.0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
-        "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
-        "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
+        "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-harfbuzz@1.91.8001@d41d8cd9",
         "reason-font-manager@2.1.1@d41d8cd9", "flex@1.2.3@d41d8cd9",
         "@reason-native/rely@3.2.1@d41d8cd9",
@@ -327,17 +327,17 @@
       ],
       "devDependencies": []
     },
-    "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9": {
-      "id": "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
+    "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9": {
+      "id": "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
       "name": "reason-skia",
-      "version": "github:revery-ui/reason-skia#2352b85",
+      "version": "github:revery-ui/reason-skia#6b459c7",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/reason-skia#2352b85" ]
+        "source": [ "github:revery-ui/reason-skia#6b459c7" ]
       },
       "overrides": [],
       "dependencies": [
-        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "esy-skia@github:revery-ui/esy-skia#07c13a9@d41d8cd9",
         "esy-sdl2@2.0.10006@d41d8cd9", "esy-freetype2@2.9.1007@d41d8cd9",
@@ -347,14 +347,14 @@
       ],
       "devDependencies": [ "@opam/dune@opam:2.5.1@a0c1e658" ]
     },
-    "reason-sdl2@2.10.3022@d41d8cd9": {
-      "id": "reason-sdl2@2.10.3022@d41d8cd9",
+    "reason-sdl2@2.10.3023@d41d8cd9": {
+      "id": "reason-sdl2@2.10.3023@d41d8cd9",
       "name": "reason-sdl2",
-      "version": "2.10.3022",
+      "version": "2.10.3023",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3022.tgz#sha1:9987104b0bad2627e66c4632e8525a177fdb8a2d"
+          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3023.tgz#sha1:ee3fea956160c9139ed588acf76c02bd80c4c962"
         ]
       },
       "overrides": [],
@@ -1179,12 +1179,12 @@
       "overrides": [],
       "dependencies": [
         "revery-terminal@github:revery-ui/revery-terminal#4e8ca1c@d41d8cd9",
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "refmterr@3.3.0@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9",
         "reason-tree-sitter@1.0.1001@d41d8cd9",
-        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "reason-libvim@github:onivim/reason-libvim#ce6b248@d41d8cd9",
         "reason-jsonrpc@1.5.0@d41d8cd9", "ocaml@4.9.0@d41d8cd9",

--- a/integrationtest.esy.lock/index.json
+++ b/integrationtest.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "41954cf4d697932cc08cce97af63383e",
+  "checksum": "74a66d9a89023d7c756b7f39c2d27ff3",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -131,20 +131,20 @@
       },
       "overrides": [],
       "dependencies": [
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "reason-libvterm@github:revery-ui/reason-libvterm#91bad1f@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#b17ce17@d41d8cd9",
         "@glennsl/timber@1.0.0@d41d8cd9"
       ],
       "devDependencies": []
     },
-    "revery@github:revery-ui/revery#e108530@d41d8cd9": {
-      "id": "revery@github:revery-ui/revery#e108530@d41d8cd9",
+    "revery@github:revery-ui/revery#9f725e0@d41d8cd9": {
+      "id": "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
       "name": "revery",
-      "version": "github:revery-ui/revery#e108530",
+      "version": "github:revery-ui/revery#9f725e0",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/revery#e108530" ]
+        "source": [ "github:revery-ui/revery#9f725e0" ]
       },
       "overrides": [],
       "dependencies": [
@@ -152,8 +152,8 @@
         "reperf@1.5.0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
-        "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
-        "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
+        "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-harfbuzz@1.91.8001@d41d8cd9",
         "reason-font-manager@2.1.1@d41d8cd9", "flex@1.2.3@d41d8cd9",
         "@reason-native/rely@3.2.1@d41d8cd9",
@@ -327,17 +327,17 @@
       ],
       "devDependencies": []
     },
-    "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9": {
-      "id": "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
+    "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9": {
+      "id": "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
       "name": "reason-skia",
-      "version": "github:revery-ui/reason-skia#2352b85",
+      "version": "github:revery-ui/reason-skia#6b459c7",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/reason-skia#2352b85" ]
+        "source": [ "github:revery-ui/reason-skia#6b459c7" ]
       },
       "overrides": [],
       "dependencies": [
-        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "esy-skia@github:revery-ui/esy-skia#07c13a9@d41d8cd9",
         "esy-sdl2@2.0.10006@d41d8cd9", "esy-freetype2@2.9.1007@d41d8cd9",
@@ -347,14 +347,14 @@
       ],
       "devDependencies": [ "@opam/dune@opam:2.5.1@a0c1e658" ]
     },
-    "reason-sdl2@2.10.3022@d41d8cd9": {
-      "id": "reason-sdl2@2.10.3022@d41d8cd9",
+    "reason-sdl2@2.10.3023@d41d8cd9": {
+      "id": "reason-sdl2@2.10.3023@d41d8cd9",
       "name": "reason-sdl2",
-      "version": "2.10.3022",
+      "version": "2.10.3023",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3022.tgz#sha1:9987104b0bad2627e66c4632e8525a177fdb8a2d"
+          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3023.tgz#sha1:ee3fea956160c9139ed588acf76c02bd80c4c962"
         ]
       },
       "overrides": [],
@@ -1179,12 +1179,12 @@
       "overrides": [ "integrationtest.json" ],
       "dependencies": [
         "revery-terminal@github:revery-ui/revery-terminal#4e8ca1c@d41d8cd9",
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "refmterr@3.3.0@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9",
         "reason-tree-sitter@1.0.1001@d41d8cd9",
-        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "reason-libvim@github:onivim/reason-libvim#ce6b248@d41d8cd9",
         "reason-jsonrpc@1.5.0@d41d8cd9", "ocaml@4.9.0@d41d8cd9",

--- a/package.json
+++ b/package.json
@@ -235,7 +235,7 @@
   "resolutions": {
     "@esy-ocaml/libffi": "onivim/libffi#590b041",
     "@opam/yojson": "onivim/yojson:yojson.opam#f480aef",
-    "revery": "revery-ui/revery#e108530",
+    "revery": "revery-ui/revery#9f725e0",
     "editor-core-types": "onivim/editor-core-types#6a8afaf",
     "esy-cmake": "prometheansacrifice/esy-cmake#2a47392def755",
     "esy-skia": "revery-ui/esy-skia#07c13a9",

--- a/test.esy.lock/index.json
+++ b/test.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "1308609526856c3295ce350527e7920f",
+  "checksum": "f0f3e44160fa3be3d32449fe5f8d97c0",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -131,20 +131,20 @@
       },
       "overrides": [],
       "dependencies": [
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "reason-libvterm@github:revery-ui/reason-libvterm#91bad1f@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#b17ce17@d41d8cd9",
         "@glennsl/timber@1.0.0@d41d8cd9"
       ],
       "devDependencies": []
     },
-    "revery@github:revery-ui/revery#e108530@d41d8cd9": {
-      "id": "revery@github:revery-ui/revery#e108530@d41d8cd9",
+    "revery@github:revery-ui/revery#9f725e0@d41d8cd9": {
+      "id": "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
       "name": "revery",
-      "version": "github:revery-ui/revery#e108530",
+      "version": "github:revery-ui/revery#9f725e0",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/revery#e108530" ]
+        "source": [ "github:revery-ui/revery#9f725e0" ]
       },
       "overrides": [],
       "dependencies": [
@@ -152,8 +152,8 @@
         "reperf@1.5.0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
-        "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
-        "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
+        "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-harfbuzz@1.91.8001@d41d8cd9",
         "reason-font-manager@2.1.1@d41d8cd9", "flex@1.2.3@d41d8cd9",
         "@reason-native/rely@3.2.1@d41d8cd9",
@@ -327,17 +327,17 @@
       ],
       "devDependencies": []
     },
-    "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9": {
-      "id": "reason-skia@github:revery-ui/reason-skia#2352b85@d41d8cd9",
+    "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9": {
+      "id": "reason-skia@github:revery-ui/reason-skia#6b459c7@d41d8cd9",
       "name": "reason-skia",
-      "version": "github:revery-ui/reason-skia#2352b85",
+      "version": "github:revery-ui/reason-skia#6b459c7",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/reason-skia#2352b85" ]
+        "source": [ "github:revery-ui/reason-skia#6b459c7" ]
       },
       "overrides": [],
       "dependencies": [
-        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "refmterr@3.3.0@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "esy-skia@github:revery-ui/esy-skia#07c13a9@d41d8cd9",
         "esy-sdl2@2.0.10006@d41d8cd9", "esy-freetype2@2.9.1007@d41d8cd9",
@@ -347,14 +347,14 @@
       ],
       "devDependencies": [ "@opam/dune@opam:2.5.1@a0c1e658" ]
     },
-    "reason-sdl2@2.10.3022@d41d8cd9": {
-      "id": "reason-sdl2@2.10.3022@d41d8cd9",
+    "reason-sdl2@2.10.3023@d41d8cd9": {
+      "id": "reason-sdl2@2.10.3023@d41d8cd9",
       "name": "reason-sdl2",
-      "version": "2.10.3022",
+      "version": "2.10.3023",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3022.tgz#sha1:9987104b0bad2627e66c4632e8525a177fdb8a2d"
+          "archive:https://registry.npmjs.org/reason-sdl2/-/reason-sdl2-2.10.3023.tgz#sha1:ee3fea956160c9139ed588acf76c02bd80c4c962"
         ]
       },
       "overrides": [],
@@ -1179,12 +1179,12 @@
       "overrides": [ "test.json" ],
       "dependencies": [
         "revery-terminal@github:revery-ui/revery-terminal#4e8ca1c@d41d8cd9",
-        "revery@github:revery-ui/revery#e108530@d41d8cd9",
+        "revery@github:revery-ui/revery#9f725e0@d41d8cd9",
         "rench@github:bryphe/rench#a976fe5@d41d8cd9",
         "refmterr@3.3.0@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9",
         "reason-tree-sitter@1.0.1001@d41d8cd9",
-        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3022@d41d8cd9",
+        "reason-textmate@3.1.2@d41d8cd9", "reason-sdl2@2.10.3023@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#ae1fd34@d41d8cd9",
         "reason-libvim@github:onivim/reason-libvim#ce6b248@d41d8cd9",
         "reason-jsonrpc@1.5.0@d41d8cd9", "ocaml@4.9.0@d41d8cd9",

